packages/orbit-components/src/Collapse/Collapse.ct.tsx

```diff
@@ -4,9 +4,19 @@ import { test, expect } from "@playwright/experimental-ct-react";
 import CollapseStory from "./Collapse.ct-story";
 
 test.describe("visual Collapse", () => {
-  test("Collapse", async ({ mount }) => {
+  test("Default collapse", async ({ mount }) => {
     const component = await mount(<CollapseStory />);
 
     await expect(component).toHaveScreenshot();
   });
+
+  test("First Collapse opens on label click and last Collapse opens on arrow button click", async ({
+    mount,
+  }) => {
+    const component = await mount(<CollapseStory />);
```

Comment:
I included visual tests of collapsed component on click for wrapper & arrow button within one test, because I wanted to avoid the huge number of screenshots.